recipe/meta.yaml

```diff
@@ -22,13 +22,13 @@ requirements:
     - python
     - setuptools
```

Comment:
I think setuptools here and in the `run` section below needs to be changed to hatchling.

Reply:
That and also adding hatch_vcs worked